### ISSUE TITLE
Strongbox doesn't get focus on Global Hot Key

### ISSUE THREAD

**Issue by daletman** [strongbox-password-safe/Strongbox#564]
When I use the Global Hot Key (⌥⌘K), Strongbox does open, but the window is not focused so I can't interact with it until I click on it. Is this intended behavior?

**Comment by strongbox-mark** [comment]
Sounds like a bug, we'll get it fixed for you in the next release.

**Comment by daletman** [comment]
How to repoduce:
1. Open iTerm2
2. Enter a command which asks for password (for me it was ssh)
3. HotKey does not work

This could be a macOS Monterey thing, since Alfreds Global Hot Keys also reproducible stop working when the user is asked for password

**Comment by strongbox-mark** [comment]
Hi @daletman 

Does this only happen when you are in the command line and a password is requested? I can't reproduce it in any other case.

**Comment by daletman** [comment]
> Hi @daletman
> 
> Does this only happen when you are in the command line and a password is requested? I can't reproduce it in any other case.

Not to my knowledge.

**Comment by strongbox-mark** [comment]
Sorry, could you clarify, have you seen this issue anywhere other than the stated command line requesting password situation?

**Comment by daletman** [comment]
Yes, I've seen it in other places, but can't reproduce reliably.

**Comment by strongbox-mark** [comment]
Hi @georgesnow @daletman - could you check if this is still an issue with 1.57.3?

**Comment by daletman** [comment]
No, focus does work fine. But in general the search field doesn't get focus when opening Strongbox with the Shortcut. Is this a settings thing?

**Comment by strongbox-mark** [comment]
OK, not sure if this is terminal related at all...

@daletman - could you check the Database > Start with Search menu item is checked?

**Comment by daletman** [comment]
> OK, not sure if this is terminal related at all...
I'm pretty sure it was a general change (or Bug) in macOS Monterey that Hotkeys did not work when a Password field was focused. I had the same issue with my Global Hotkey for Alfred. I couldn't trigger it when any Password field was focused (or the Terminal prompt was asking for a password)

> @daletman - could you check the Database > Start with Search menu item is checked?
Ah yes, now it works, thanks you!

**Comment by strongbox-mark** [comment]
Alright, I'll give @georgesnow a chance to comment but I think this is probably one to close soon.

**Comment by georgesnow** [comment]
Hmm I did a quick test after doing the strongbox update. It doesn’t seem to work for me from iterm when there is password prompt within the terminal window.

however I tried 3 other apps with global hot keys same deal. The hot keys don’t work. This seems specific to terminals and password prompts. I will have to test it further to confirm.

**Comment by daletman** [comment]
@georgesnow which macOS version are you using?

**Comment by strongbox-mark** [comment]
I think I'm going to close this one out guys, seems like a weird OS level edge case.

**Comment by djpowers** [comment]
Chiming in late, but I just happened upon a similar issue recently where no apps (including Strongbox) would get focus when I tried to trigger their global hot keys while iTerm2 was the active application. I was seeing the same described behavior regardless of whether the terminal was displaying a password prompt.

For me, turning off "Secure Keyboard Entry" (under the iTerm2 menu item) fixed the problem. More info here: https://iterm2.com/documentation/2.1/documentation-menu-items.html
